webapp/components/suggestion/at_mention_provider.jsx

```diff
@@ -112,9 +112,10 @@ export default class AtMentionProvider {
     handlePretextChanged(suggestionId, pretext) {
         clearTimeout(this.timeoutId);
 
-        const captured = (/@([a-z0-9\-\._]*)$/i).exec(pretext.toLowerCase());
-        if (captured) {
-            const prefix = captured[1];
+        const captured = (/(\w*)@([a-z0-9\-\._]*)$/i).exec(pretext.toLowerCase());
```

Comment:
You can actually make this a bit simpler by changing the regex to `/(?:^|\W)@([a-z0-9\-\/_]*)*/i`.  That way, it's a bit clearer that it's looking for an at mention that's either at the start of the text or following any non-word character, and the `?:` will mean the `const prefix = captured[1];` line below won't need to change.